Jenkinsfile

```diff
@@ -0,0 +1,110 @@
+
+pipeline {
+  environment {
+    DOCKER_NETWORK = ''
+  }
+  options {
+    skipDefaultCheckout()
+    buildDiscarder(logRotator(numToKeepStr: '20'))
+    timestamps()
+  }
+  agent any
+  stages {
+    stage('Stop same job builds') {
+      agent { label 'master' }
+      steps {
+        script {
+          def scmVars = checkout scm
+          // need this for develop->master PR cases
+          // CHANGE_BRANCH is not defined if this is a branch build
+          try {
+            scmVars.CHANGE_BRANCH_LOCAL = scmVars.CHANGE_BRANCH
+          }
+          catch (MissingPropertyException e) {
+          }
+          if (scmVars.GIT_LOCAL_BRANCH != "develop" && scmVars.CHANGE_BRANCH_LOCAL != "develop") {
+            def builds = load ".jenkinsci/cancel-builds-same-job.groovy"
```

Comment:
This file is missing. Either remove the entire `if` block, or copy the file from the old repo.

Reply:
https://github.com/d3ledger/d3-btc/blob/feature/codeMigration/.jenkinsci/cancel-builds-same-job.groovy